client/src/components/NavLinks/NavLinks.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import HomeIcon from '../Icons/HomeIcon/HomeIcon';
```

Comment:
Check this out: https://create-react-app.dev/docs/importing-a-component#absolute-imports
This lets you configure aliases for the build system.
You could do something like:
`import HomeIcon from 'components/Icons/HomeIcon/HomeIcon`

Reply:
awesome!!!!!